docs/stable/1-client.md

```diff
@@ -94,33 +96,33 @@ To communicate between Status nodes, the [RLPx Transport
 Protocol, v5](https://github.com/ethereum/devp2p/blob/master/rlpx.md) is used, which
 allows for TCP-based communication between nodes.
 
-On top of this we run the RLPx-based subprotocol [Whisper
-v6](https://eips.ethereum.org/EIPS/eip-627) for privacy-preserving messaging.
-
-There MUST be a node that is capable of discovering peers and
-implements Whisper V6 specification.
+On top of this we run RLPx-based subprotocols, the client
+MAY use [Whisper v6](https://eips.ethereum.org/EIPS/eip-627), though the client 
+SHOULD use [Waku V1](https://github.com/vacp2p/specs/blob/master/specs/waku/waku-1.md)
+for privacy-preserving messaging.
 
 #### Node discovery and roles
 
-There are four types of node roles:
-1. Bootstrap nodes
-2. Whisper relayers
-3. Mailservers (servers and clients)
-4. Mobile nodes (Status Clients)
+There are five types of node roles:
+1. `Bootstrap node`
+1. `Whisper/Waku relayer`
+1. `Mailserver` (servers and clients)
+1. `Mobile node` (Status Clients)
 
-To implement a standard Status client you MUST implement both 2. and 4. node types. The
-other node types are optional, but we RECOMMEND you implement a mailserver
-client mode, otherwise the user experience is likely to be poor.
+A standard Status client MUST implement `Whisper/Waku relayer` roles,
```

Comment:
Hmm I realize this was in old code, but is this true? If you implement a mobile node, do you also need to implement a relay node? It is true we get forwarded envelopes on mobile (but don't relay further). What do we mean by mobile node in that case? Something seems off here

cc @kdeme @cammellos @decanus

Reply:
Yeah, I guess it should rather be something like:
"A Status client MUST implement the `Mobile node` role and MAY implement `Whisper/Waku relayer` and `Mailserver` roles."

Reply:
The whole role thing is not so clear though. I think it is because we are mixing Whisper/Waku & Status Client layers. And then also bootstrap..

Reply:
>If you implement a mobile node, do you also need to implement a relay node?

I puzzled over this question when I was drafting this PR.

>"A Status client MUST implement the Mobile node role and MAY implement Whisper/Waku relayer and Mailserver roles."

Are there circumstances when a Status node would be active an not be using p2p messaging?

Reply:
Yeah, we need to clarify the roles a lot more. Since this issue exists in previous spec, could you open an issue for clarifying this @Samyoul? Then we can deal with it separately to get this merged. (I think there was some related roles issue but can't find it right now)

Reply:
https://github.com/status-im/specs/issues/123